Assets/Packages/com.studio23.ss2.audiosystem.fmod/Runtime/Core/MixerManager.cs

```diff
@@ -0,0 +1,125 @@
+using Cysharp.Threading.Tasks;
+using Studio23.SS2.AudioSystem.fmod.Data;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.CompilerServices;
+
+[assembly: InternalsVisibleTo("com.studio23.ss2.audiosystem.fmod.playmode.tests")]
+namespace Studio23.SS2.AudioSystem.fmod.Core
+{
+    public class MixerManager
+    {
+        internal List<FMODBusData> _busDataList;
```

Comment:
```suggestion
        internal List<FMODBusData> _busDataList;
```

This should be a dictionary of key value pairs please make methods defined bellow adapt to the changes @coderabbitai would you please review this MixerManager.cs file and suggest changes. for example the lists should be dictonaries